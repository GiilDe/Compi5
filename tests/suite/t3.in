void main(){
    int x = ((2));
    int z = 5;
    int y = 2*z/x;

    printi(z);
    z = (5)/(x-2);
    print("Im mean!");
}
