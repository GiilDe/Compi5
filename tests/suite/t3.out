5Error division by zero
